index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="index.css">
+    <title>Portifolio page revire</title>
+</head>
+
+<body>
+    <nav id="navbar">
+        <img id="logo" class="dagi" width='150'
+            src="https://avatars.githubusercontent.com/u/68595873?s=400&u=80bbdfc291f01cffaae57bee393b989ea3b24bda&v=4"
+            alt="">
+
```

Comment:
Add link to the logo

Reply:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes 😎

Also, please be sure to tell your requirements/suggestions in a way that it sounds like suggestions instead of orders 😉 The idea is to make it politely and friendly 😬